### PR TITLE
Chapter/chapter3 organizing your code

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, I write code for chapter 3 in ebook write-idiomatic-python

### CODE REVIEW

File: organizing_your_code/imports/contants_of_module.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- Do not use from foo import * to import the contents of a module.
+- Instead, group import statements within parentheses or use absolute imports
+with an "as" clause to keep your namespace clean and organized.
+"""
+
+
+# Idiomatic solution
+from foo import (bar, baz, qux,
+        quux, quuux)
+import foo
```
Comment (cuongdoan-agilityio) [inline organizing_your_code/imports/contants_of_module.py]: Please check again, you are importing twice.

File: organizing_your_code/executable_scripts/division_of_two_numbers.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    return first_parameter / second_parameter
+
+# Will only run if script is executed directly,
+# not when the file is imported as a module
+
+
+if __name__ == '__main__':
+    first_number = float(sys.argv[1])
+    second_number = float(sys.argv[2])
```
Comment (cuongdoan-agilityio) [inline organizing_your_code/executable_scripts/division_of_two_numbers.py:53]: Please double check, the parameters type on docstring is int.

File: organizing_your_code/executable_scripts/proper_error_codes.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    # Again, what if this is False? How would other
+    # programs know?
+
+    # if result:
+    #     do_stuff_with_result(result)
+
+
+# Idiomatic solution
+def do_stuff(argument):
```
Comment (cuongdoan-agilityio) [inline organizing_your_code/executable_scripts/proper_error_codes.py:29]: Missing docstring.

File: organizing_your_code/executable_scripts/division_of_two_numbers.py (hunk 2 above)
Comment (hothanhtuan1209) [reply organizing_your_code/executable_scripts/division_of_two_numbers.py:53]: Done

File: organizing_your_code/executable_scripts/proper_error_codes.py (hunk 3 above)
Comment (hothanhtuan1209) [reply organizing_your_code/executable_scripts/proper_error_codes.py:29]: Done

File: organizing_your_code/imports/contants_of_module.py (hunk 1 above)
Comment (hothanhtuan1209) [reply organizing_your_code/imports/contants_of_module.py]: Done

**Pull request merged**